components/scream/docs/build.md

```diff
@@ -0,0 +1,93 @@
+# Building and Testing Standalone SCREAM
+
+Follow these simple instructions to build and test SCREAM's standalone
+configuration for yourself. This document makes use of the following
+paths:
+
++ `${RUN_ROOT_DIR}`: the root directory where SCREAM is built and run
++ `${SCREAM_SRC_DIR}`: the directory into which you've cloned the `scream` repo
```

Comment:
Is this the repo root dir, or is it components/scream? I think you mean the latter, but your description makes it sound like the git repo root dir.

Reply:
It's supposed to be the repo root directory. I think I've fixed references to it.